packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -31,141 +32,150 @@ However, if an error occurs during sign-in, Okta may need to display an error pa
+   * Click **Revert changes** to remove your customizations and restore the default HTML/CSS and JavaScript code.
 
-1. In the Admin Console, select **Customizations** and then **Error pages code editor**.
-1. Make changes directly in the embedded **Error Pages** editor.
-1. Click **Preview** to preview your changes before you publish.
-1. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
-1. Click **Save and Publish** to commit your changes.
+   > **Note:** The Admin Console auto-saves your changes every few seconds. You can navigate away from the code editor and your updates remain. To discard your changes without publishing them, turn off the toggle next to **Code editor**.
 
-> **Note:** See [Customization examples](#customization-examples) for snippets that you can update and use.
+5. Click **Publish** to commit your changes.
 
-## Use macros
+## Use variables
 
-The following macros contain the configuration parameters for certain page elements. These macros inject specific content or functionality automatically.
+The following variables contain the configuration parameters for certain page elements. These variables inject specific content or functionality automatically.
 
-### <span v-pre>`{{orgName}}`</span>
+> **Note:** Variables with double curly braces return escaped HTML by default. Triple braces `{{{` are used for the `errorDescription` macro to return unescaped HTML. See [Mustache template](http://mustache.github.io/mustache.5.html).
 
-Inserts the org name title.
+### orgName
+
+Inserts the org name title. Requires double brackets: <span v-pre>`{{orgName}}`</span>
 
 Example:
 
 ```html
 <title>{{orgName}} - {{errorSummary}}</title>
 ```
 
-### <span v-pre>`{{errorSummary}}`</span>
+### errorSummary
 
-Inserts the error title text.
+Inserts the error title text. Requires double curly braces: <span v-pre>`{{errorSummary}}`</span>
 
 Example:
 
 ```html
 <h2 class="o-form-title">{{errorSummary}}</h2>
 ```
 
-### <span v-pre>`{{bgImageUrl}}`</span>
+### bgImageUrl
+
+Inserts a URL to the background image configured for your application. Requires double curly braces: <span v-pre>`{{bgImageUrl}}`</span>
 
-Inserts a URL to the background image configured for your application. You can change this image by using the **Sign-in Configuration** option that is accessed by selecting **Settings**, and then **Appearance** from the Admin Console, but this changes the background image in all instances where the macro is used, including your custom sign-in page.
+You can change this image by using the [Sign-in Configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext_Settings_Appearance) option, but this changes the background image in all instances where the macro is used, including your custom sign-in page.
```

Comment:
this link is now obsolete https://help.okta.com/okta_help.htm?type=oie&id=ext_Settings_Appearance

This is the new one https://help.okta.com/oie/en-us/Content/Topics/Settings/branding-set-theme.htm

also replace `macro` with `variable`

Reply:
Thanks, Albert. There's no alias for that topic yet, so I just submitted a Jira. Shouldn't take long to get it.

Reply:
Done.